tests/strategies/test_deepspeed_strategy.py

```diff
@@ -846,10 +844,10 @@ def on_train_batch_start(
     model = ModelParallelClassificationModel()
     trainer = Trainer(
         default_root_dir=tmpdir,
-        fast_dev_run=True,
         strategy=DeepSpeedStrategy(stage=3),
         accelerator="gpu",
         devices=1,
+        max_epochs=2,
```

Comment:
Since fast_dev_run was removed, we need to limit the # of batches to reduce the runtime

```suggestion
        max_epochs=2,
        limit_train_batches=1,
        limit_val_batches=0,
```